AOEMods.Essence.Editor/ArchiveViewModel.cs

```diff
@@ -37,6 +29,9 @@ public ArchiveViewModel()
             PackCommand = new RelayCommand(Pack);
             UnpackCommand = new RelayCommand(Unpack);
             OpenPropertiesCommand = new RelayCommand(OpenProperties);
+            ExpandCollapseAllCommand = new RelayCommand(ExpandCollapseAll);
+            IsExpanded = false;
+            SearchText = "";

```

Comment:
Set this on isExpanded and searchText directly (or is there a reason it needs to be here like triggering something?)